src/main/sensors/gyro.h

```diff
@@ -201,8 +201,8 @@ typedef struct gyroConfig_s {
 
     uint8_t  gyro_filter_debug_axis;
 
-    uint8_t gyrosDetected; // What gyros should detection be attempted for on startup. Automatically set on first startup.
-    uint8_t dyn_lpf_curve_expo; // set the curve for dynamic gyro lowpass filter
+    uint8_t  gyrosDetected; // What gyros should detection be attempted for on startup. Automatically set on first startup.
```

Comment:
This is going the wrong way - single space should be used.

Reply:
Changed all of them to single use a single space.